internal/provisioner/cluster.go

```diff
@@ -30,15 +30,20 @@ func CreateCluster(provider, size string, logger log.FieldLogger) error {
 		return fmt.Errorf("unsupported size %s", size)
 	}
 
-	outputDir := "tmp"
-	_, err := os.Stat("tmp")
+	if clusterId == "" {
+		clusterId = model.NewId()
+	}
+
+	// Temporarily relocate the kops output directory to a local folder based on the
```

Comment:
Nit: at this point, we're not "relocating" per se (made more sense in the immediate vicinity of `os.Rename`).

```suggestion
	// Temporarily locate the kops output directory to a local folder based on the
```